stack/.env

```
@@ -23,4 +23,7 @@ WLS_INFOMANAGEMENT_SERVICE_KEYCLOAK_URL=http://kubernetes.docker.internal:8100/a
 
 WLS_BRIEFWAHL_SERVICE_PROFILE=local,db-oracle
 WLS_BRIEFWAHL_SERVICE_DB_URL=jdbc:oracle:thin:@//wls-db-oracle:1521/XEPDB1
-WLS_BRIEFWAHL_SERVICE_KEYCLOAK_URL=http://kubernetes.docker.internal:8100/auth/realms/${SSO_REALM}/protocol/openid-connect
\ No newline at end of file
+WLS_BRIEFWAHL_SERVICE_KEYCLOAK_URL=http://kubernetes.docker.internal:8100/auth/realms/${SSO_REALM}/protocol/openid-connect
```

Analog zu den anderen Configs eine Leerzeile zur besseren Lesbarkeit

```suggestion

WLS_BRIEFWAHL_SERVICE_KEYCLOAK_URL=http://kubernetes.docker.internal:8100/auth/realms/${SSO_REALM}/protocol/openid-connect
```